util/file.go

```diff
@@ -46,6 +46,16 @@ func CanonicalPaths(paths []string, basePath string) ([]string, error) {
 	return canonicalPaths, nil
 }
 
+// Delete the given list of files
+func DeleteFiles(files []string) error {
+	for _, file := range files {
+		if err := os.Remove(file); err != nil {
+			return errors.WithStackTrace(err)
+		}
+	}
+	return nil
+}
```

Comment:
It's designed solely to delete files. I've updated the comment to make that clearer: https://github.com/gruntwork-io/terragrunt/pull/114/commits/931215de4b4a61f043ca735767af497ea949a93f